admin_manual/configuration_server/background_jobs_configuration.rst

```diff
@@ -64,17 +64,17 @@ Using the operating system cron feature is the preferred method for executing
 regular tasks.  This method enables the execution of scheduled jobs without the
 inherent limitations the Web server might have.
 
-To run a cron job on a \*nix system, every 15 minutes, under the default Web
+To run a cron job on a \*nix system, every 5 minutes, under the default Web
 server user (often, ``www-data`` or ``wwwrun``), you must set up the following
 cron job to call the **cron.php** script::
 
   # crontab -u www-data -e
-  */15  *  *  *  * php -f /var/www/nextcloud/cron.php
+  */5  *  *  *  * php -f /var/www/nextcloud/cron.php
 
 You can verify if the cron job has been added and scheduled by executing::
 
   # crontab -u www-data -l
-  */15  *  *  *  * php -f /var/www/nextcloud/cron.php
+  */5  *  *  *  * php -f /var/www/nextcloud/cron.php
```

Comment:
@rullzer @nickvergessen @schiessle @blizzz I used this quite often at instances and it just makes the overall speed of email deliveries for example just faster. Thus I changed it here as well. Just that you are aware of it ;)

Reply:
Do the same on my instance :+1:

Reply:
I have 5 minutes since ever.
Maybe we should also recommend this in the admin settings/whats new